Polyhedron/demo/Polyhedron/Plugins/Mesh_3/raw_image.ui

```diff
@@ -1,62 +1,81 @@
-<ui version="4.0" >
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
  <class>Raw_image_dialog</class>
- <widget class="QDialog" name="Raw_image_dialog" >
-  <property name="geometry" >
+ <widget class="QDialog" name="Raw_image_dialog">
+  <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>585</width>
     <height>342</height>
    </rect>
   </property>
-  <property name="windowTitle" >
+  <property name="windowTitle">
    <string>Open raw image</string>
   </property>
-  <layout class="QVBoxLayout" name="verticalLayout" >
+  <layout class="QVBoxLayout" name="verticalLayout">
    <item>
-    <widget class="QGroupBox" name="groupBox_2" >
-     <property name="title" >
+    <widget class="QGroupBox" name="groupBox_2">
+     <property name="title">
       <string>Image &amp;value type</string>
      </property>
-     <layout class="QGridLayout" name="gridLayout" >
-      <item row="2" column="0" >
-       <widget class="QRadioButton" name="short_bt" >
-        <property name="text" >
+     <layout class="QGridLayout" name="gridLayout">
+      <item row="2" column="0">
+       <widget class="QRadioButton" name="short_bt">
+        <property name="toolTip">
+         <string>Word kind of the image : short int.</string>
+        </property>
+        <property name="text">
          <string>Short (16 bits)</string>
         </property>
```

Comment:
I do not see the need for a tooltip. Just change the text of the button to "Short int (16 bits)" and that will add the same information.